WalletWasabi/Helpers/PasswordHelper.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace WalletWasabi.Helpers
+{
+	public static class PasswordHelper
+	{
+		public static string[] GetPossiblePasswords(string originalPassword)
+		{
+			List<string> possiblePasswords = new List<string>()
+			{
+				originalPassword,
+				StringCutIssue(originalPassword)
+			};
+
+			return possiblePasswords.ToArray();
+		}
+
+		private static string StringCutIssue(string text)
+		{
+			// On OSX Avalonia gets the string from the Clipboard as byte[] and size.
+			// The size was mistakenly taken from the size of the original string which is not correct because of the UTF8 encoding.
+
+			byte[] bytes = Encoding.Default.GetBytes(text);
```

Comment:
We cannot know what encoding is `Encoding.Default`. Two computers can use two different encodings as default so, the same password can be decoded differently. I would prefer to make the encoding explicit.

Reply:
But if the system default is used to encode/decode the clipboard content per OP system then this will be the good solution.

Reply:
According to your suggestion and [this](https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding.default?view=netcore-2.2)
I changed it to UTF8